src/Evolve/Evolve.cs

```diff
@@ -397,7 +398,22 @@ private void InternalMigrate(DatabaseHelper db)
             }
             else
             {
```

Comment:
Since the `using` was removed the scope will never be disposed nor the timeouts reverted if something throws. I think a try-finally should be used here;
 ```suggestion
                 var defaultAmbientTransactionTimeout = TransactionManager.DefaultTimeout;

                TransactionScope scope = null;
                if (this.AmbientTransactionTimeout != null) {
                    var newAmbientTransactionTimeout = new TimeSpan(0, 0, this.AmbientTransactionTimeout.Value);
                    ConfigureTransactionTimeoutCore(newAmbientTransactionTimeout);

                    scope = new TransactionScope(TransactionScopeOption.Required, newAmbientTransactionTimeout);

                } else {
                    scope = new TransactionScope();
                }
                try {
                    db.WrappedConnection.UseAmbientTransaction();
                    lastAppliedVersion = Migrate();

                    if (TransactionMode == TransactionKind.CommitAll) {
                        scope.Complete();
                    } else {
                        LogRollbackAppliedMigration();
                    }
                }
                finally {
                    scope.Dispose();

                    if (this.AmbientTransactionTimeout != null) {
                        ConfigureTransactionTimeoutCore(defaultAmbientTransactionTimeout);
                    }
                }
```
I think this would fix the failing tests as well.